api/src/main/java/com/couponPayment/service/PaymentService.java

```diff
@@ -0,0 +1,144 @@
+package com.couponPayment.service;
+
+import com.couponPayment.consts.CommonResult;
+import com.couponPayment.consts.PaymentStatus;
+import com.couponPayment.dto.ApiResponse;
+import com.couponPayment.dto.PaymentCancelReq;
+import com.couponPayment.dto.PaymentCancelRes;
+import com.couponPayment.dto.PaymentReq;
+import com.couponPayment.dto.PaymentRes;
+import com.couponPayment.dto.TossBillingPaymentCancelReq;
+import com.couponPayment.dto.TossBillingPaymentCancelRes;
+import com.couponPayment.dto.TossBillingPaymentReq;
+import com.couponPayment.dto.TossBillingPaymentRes;
+import com.couponPayment.entity.MyWalletInfo;
+import com.couponPayment.entity.StoreInfo;
+import com.couponPayment.entity.TransactionInfo;
+import com.couponPayment.entity.UserInfo;
+import com.couponPayment.entity.dto.TransactionInfoDto;
+import com.couponPayment.entity.mapper.TransactionMapper;
+import com.couponPayment.repository.MyWalletInfoRepository;
+import com.couponPayment.repository.TransactionInfoRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+@Service
+@RequiredArgsConstructor
+public class PaymentService {
+
+    private final MyWalletInfoRepository myWalletInfoRepository;
+    private final TransactionInfoRepository transactionInfoRepository;
+    private final TransactionMapper transactionMapper;
+    private final TossBillingService tossBillingService;
+
+    @Transactional
+    public ApiResponse<PaymentRes> payment(PaymentReq paymentReq) {
+        MyWalletInfo myWalletInfo = myWalletInfoRepository.findByCardIdAndUseFlag("cardId", 0).get();
+        UserInfo userInfo = myWalletInfo.getUserInfo();
+        StoreInfo storeInfo = userInfo.getStoreInfo();
+
+        String tossPaymentId = storeInfo.getTossPaymentId();
+        String customerKey = userInfo.getCustomerKey();
+
+        LocalDateTime now = LocalDateTime.now();
+        String formattedDate = now.format(DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss"));
```

Comment:
```suggestion
        String formattedDate = now.format(DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss"));
```

이 부분은 Pattern을 미리 만들어서 재사용하는 것이 좋지 않을까요? 아니면, ofPattern은 같은걸 계속 재사용하는 코드가 있는 걸까요? :-)